language/src/main/java/com/oracle/truffle/sl/SLLanguage.java

```diff
@@ -190,7 +190,7 @@
  * variables.
  * </ul>
  */
-@TruffleLanguage.Registration(id = SLLanguage.ID, name = "SL", version = "0.30", mimeType = SLLanguage.MIME_TYPE)
+@TruffleLanguage.Registration(id = SLLanguage.ID, name = "SL", version = "1.0", mimeType = SLLanguage.MIME_TYPE)
```

Comment:
You can ommit the language version now with rc2. This change should really be made in the graal repo.